### ISSUE TITLE
🐞 Bug: breadcrumb > active page is missing attribute aria-current="page"

### ISSUE THREAD

**Issue by MarcusMorba** [public-ui/kolibri#7159]
### Which component/area(s) are affected? (Select all that apply)
Breadcrumb

### To Reproduce
Check source code of produced HTML markup.

### Link to the code that reproduces this issue:
https://public-ui.github.io/docs/components/breadcrumb

### Can you categorise where the error occurs?
active page-indicator: li-element

### Which browser or operating system do you used to test KoliBri?
Chrome latest / Win 11 Pro

### How to reproduce issue?
Check source code of produced HTML markup.

### Current vs. Expected:
Current: at the moment, the selected page has no "aria-current"-attribute.
 Expected: set attribute "aria-current=page"-attribute. Example, please see https://designsystem.digital.gov/components/breadcrumb/

### Environment informations:
```bash Not relevant ```

**Comment by MarcusMorba** [comment]
Please also have a look at the latest WCAG 2.2 Gov.uk-example: https://design-system.service.gov.uk/components/breadcrumbs/#wcag-interact-breadcrumbs

**Comment by deleonio** [comment]
Hi @MarcusMorba,

thank you very much for this topic.

Actually for this case we provide this: https://public-ui.github.io/docs/components/link#aria-current-service

This method manages the state for all links at the page.

We will check the topic, if we can do more.

**Comment by deleonio** [comment]
1. Beispiele optimieren bzw. ein konkretes anbieten.
2. Bei Breadcrumb können wir ja eine Annahme treffen, oder?

**Comment by laske185** [comment]
Beim Breadcrumb wird im letzten `li`-Element (aktuelle Seite) im `span` `ara-current="page"` gesetzt.

Referenzen:
- Spezifikation: https://w3c.github.io/aria/#aria-current 
- W3C: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/

**Comment by deleonio** [comment]
Hint: Aria-Current kann an jeden beliebigen Tag verwendet werden. Bei KoliBri ist die aktuelle Seite kein Link, sondern nur eine fett hervorgehobener Text.

Hier noch weiteren Input von @cbraehmig: https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/

Wir könnten das auch ändern, jedoch ist die aktuelle Umsetzung der KolBreadcrumb nach w3c auch okay. Hast Du (@MarcusMorba) ggf. noch Input für uns hierzu.

**Comment by deleonio** [comment]
Bitte so umsetzen ... `aria-current` am `span` für aktuelle Seite setzen.

<img width="1498" height="604" alt="Image" src="https://github.com/user-attachments/assets/3cc6874b-4802-4ffa-ae71-7af8b768486e" />
